backend/index.js

```diff
@@ -96,17 +97,28 @@ app.post("/login", async (req, res) => {
   }
 });
 
-// Protected route example
-app.get("/protected", authenticateToken, async (req, res) => {
-  try {
-    const user = await prisma.user.findUnique({ where: { id: req.user.id } });
-    if (!user) {
-      return res.status(401).json({ message: "User not found" });
+//Endpoint for fethcing businesses
+app.get("/api/businesses", (req, res) => {
+  const jsonFilePath = path.join(
+    __dirname,
+    "public",
+    "businessApiResponse.json"
+  );
+
+  fs.readFile(jsonFilePath, "utf8", (err, data) => {
+    if (err) {
+      console.error("Error reading JSON file:", err);
+      return res.status(500).json({ error: "Failed to read JSON file" });
     }
-    res.status(200).json(user);
-  } catch (error) {
-    res.status(401).json({ message: "Token is not valid" });
-  }
+
+    try {
+      const jsonData = JSON.parse(data);
+      res.json(jsonData);
```

Comment:
this seems like we're parsing the JSON string (line 115) and then stringifying the JSON back (line 116), which seems redundant at first glance. Might be worth a comment saying something like `// Validate the JSON encoding`